Tasks/AzureWebAppContainerV1/taskparameters.ts

```diff
@@ -36,6 +39,18 @@ export class TaskParametersUtility {
         var endpointTelemetry = '{"endpointId":"' + taskParameters.connectedServiceName + '"}';
         console.log("##vso[telemetry.publish area=TaskEndpointId;feature=AzureRmWebAppDeployment]" + endpointTelemetry);
 
+        taskParameters.isMultiContainer = false;
+        taskParameters.isMultiContainer = taskParameters.ImageName && taskParameters.ImageName.indexOf("\n") !=-1;
+        tl.debug(`is multicontainer app : ${taskParameters.isMultiContainer}`);
+
+        if(taskParameters.isMultiContainer)
+        {
+            PackageUtility.getPackagePath(taskParameters.ConfigFilePath);
```

Comment:
`taskParameters.ConfigFilePath = PackageUtility.getPackagePath(taskParameters.ConfigFilePath);`

Reply:
It is done.